specifyweb/frontend/js_src/lib/components/Atoms/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -73,6 +50,17 @@ exports[`Button.Icon renders without errors 1`] = `
 </DocumentFragment>
 `;
 
+exports[`Button.Info renders without errors 1`] = `
+<DocumentFragment>
+  <button
+    class="button rounded cursor-pointer active:brightness-80 px-4 py-2
+    disabled:bg-gray-200 disabled:text-gray-500 dark:disabled:!bg-neutral-700 gap-2
+    inline-flex items-center capitalize justify-center shadow-sm dialog-icon-info hover:brightness-150 bg-[color:var(--info-button-color)] dark:bg-[color:var(--info-button-color-dark)] text-white"
```

Comment:
instead of `bg-[color:var(--info-button-color)] dark:bg-[color:var(--info-button-color-dark)]` here, modify ApplyPreferences.tsx to get the value of the `--info-button-color` CSS variable from either dark more or light mode user pref depending on the result of `useDarkMode()`

This way, if you need to use `--info-button-color` in multiple places, you don't have to provide separate dark mode styles in every place.

Do the same for all the other colors you added. If you need an example, take a look at how this was done for other colors in ApplyPreferences.tsx

Reply:
example for you from ApplyPreferences.tsx:
```
  const background = darkMode ? prefs.darkBackground : prefs.background;
  React.useEffect(
    () => document.body.style.setProperty('--background', background),
    [background]
  );
```